Resources/Private/Scripts/ColorPickerEditor/src/NeosColorPicker/Hue.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component, PureComponent } from 'react'
+import reactCSS from 'reactcss'
+import * as hue from 'react-color/lib/helpers/hue'
+
+export class Hue extends (PureComponent || Component) {
+	componentWillUnmount() {
+		this.unbindEventListeners()
+	}
+	
+	handleChange = (e) => {
+		const change = hue.calculateChange(e, this.props.direction, this.props.hsl, this.container)
```

Comment:
I get an error `Cannot read property 'h' of undefined` here when using the `hsla` mode and selecting a color from the rgb and alpha bars. Selections from the other fields work fine.

Reply:
Sorry for the bad testing. I added some last minute style changes and didn't check the functionality afterwards.